### DISCUSSION TITLE
Room Cleaning Set up?

### DISCUSSION THREAD

**Discussion by sewshoetz** [homebridge-xiaomi-roborock-vacuum/homebridge-xiaomi-roborock-vacuum#504] [Q&A]
Hello, first of all: Thanks for the great Plugin!!

But I have an issue i´m not able to solve:
In my Xiaomi App i have defined the "Küche" (Kitchen) as separate room and it works fine starting it from the xiaomi app.
I have it defined as "Küche" as well in the plugin, and the robo starts from the dock, and when he´s off the dock, he says:
Room Cleaning Completed
without doing any cleaning at all
and returns to the dock.

How do i have to set up the room cleaning?
How do i have to define it in the plug in, and whats the number before the room name?
Can someone help me fix the issue?
Thanks in advance!!
Johannes

**Comment by YoeriVD** [comment]
This may only be applicable to my case.

When I had this issue, the room was virtually unreachable. Meaning that the last time the robot tried to reach that room, it couldn't (eg. because of a closed door). At the end of that run it will save the map and remember that it can't reach it.

It is easy to verify if this is the case. Make sure every room is reachable and run a full cleaning.
The room command should work again after it finishes the map saving.

The workaround I applied is to manually change my room mapping in the Xiaomi app, so the room extends slightly beyond the door. Enough to be "reachable".
It has been working as it should ever since. I hope this helps.


![image](https://github.com/homebridge-xiaomi-roborock-vacuum/homebridge-xiaomi-roborock-vacuum/assets/2865793/f4c16f3b-637e-429b-95a4-8692cb1e5b82)
